scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -134,7 +140,18 @@ ruleTester.run('protractor-practices', rule, {
       code:
       `genderList.filter(function() {
         gender = 'female';
-      })`,
+      });`,
+      errors: [{
+        message: 'Please do not use .filter(), consider using a for loop',
```

Comment:
nit: since "consider using a for loop" is a full independent clause, you need a semicolon instead of a comma

```suggestion
        message: 'Please do not use .filter(); consider using a for loop',
```